app/native/app/(tabs)/couchers.tsx

```diff
@@ -0,0 +1,146 @@
+import { Image, StyleSheet, Button, Text } from "react-native";
+
+import ParallaxScrollView from "@/components/ParallaxScrollView";
+import { ThemedText } from "@/components/ThemedText";
+import { ThemedView } from "@/components/ThemedView";
+import { useRef, useState } from "react";
+import { useMutation } from "react-query";
+import { RpcError } from "grpc-web";
+import { service } from "service";
+import { StatusRes } from "proto/bugs_pb";
+import Alert from "components/Alert";
+import LoginForm from "features/auth/login/LoginForm";
+
+import { useAuthContext } from "features/auth/AuthProvider";
+
+import { useTranslation } from "i18n";
+import { WebView, WebViewNavigation } from "react-native-webview";
+import { AUTH, GLOBAL } from "i18n/namespaces";
+
+const WEB_BASE_URL = process.env.EXPO_PUBLIC_WEB_BASE_URL!;
+
+export default function CouchersScreen() {
+  const { t } = useTranslation([AUTH, GLOBAL]);
+
+  const [pressed, setPressed] = useState<number>(0);
+
+  const { authState, authActions } = useAuthContext();
+
+  let webview = useRef<WebView>(null);
+
+  const { data: resp, mutate: checkCoucherCount } = useMutation<
+    StatusRes.AsObject,
+    RpcError,
+    void
+  >(() => service.version.status());
+
+  const logIn = () => {
+    authActions.passwordLogin({
+      username: "aapeli_native",
+      password: "unsafepassword",
+      rememberDevice: true,
+    });
+  };
```

Comment:
I mean I know this is probably temporary, but not sure you want a record of the password in the git history?